mne/io/cnt/cnt.py

```diff
@@ -0,0 +1,230 @@
+"""Conversion tool from Neuroscan CNT to FIF
+"""
+
+# Author: Jaakko Leppakangas <jaeilepp@student.jyu.fi>
+#
+# License: BSD (3-clause)
+from os import path
+import datetime
+import calendar
+
+import numpy as np
+
+from ...utils import warn, verbose
+from ..constants import FIFF
+from ..utils import _mult_cal_one
+from ..meas_info import _empty_info
+from ..base import _BaseRaw
+
+
+def read_raw_cnt(input_fname, read_blocks=True, preload=False, verbose=None):
+    """
+
+    Parameters
+    ----------
+    input_fname : str
+        Path to the data file.
+    read_blocks : bool
+        Whether to read data in blocks. This is for dealing with different
+        kinds of CNT data formats.
+    preload : bool or str (default False)
+        Preload data into memory for data manipulation and faster indexing.
+        If True, the data will be preloaded into memory (fast, requires
+        large amount of memory). If preload is a string, preload is the
+        file name of a memory-mapped file which is used to store the data
+        on the hard drive (slower, requires less memory).
+    verbose : bool, str, int, or None
+        If not None, override default verbose level (see mne.verbose).
+
+    Returns
+    -------
+    Instance of RawCNT.
+    """
+    return RawCNT(input_fname, read_blocks=read_blocks, preload=preload,
+                  verbose=verbose)
+
+
+def _get_cnt_info(input_fname, read_blocks):
+    """Helper for reading the cnt header."""
+    dtype = '<i4'
+    n_bytes = np.dtype(dtype).itemsize
+    offset = 900  # Size of the 'SETUP' header.
+    start = 0
+    cnt_info = dict()
+    # Reading only the fields of interest. Structure of the whole header at
+    # http://paulbourke.net/dataformats/eeg/
+    with open(input_fname, 'rb', buffering=0) as fid:
+        fid.seek(225)
+        session_date = ''.join(np.fromfile(fid, dtype='S1', count=10))
+        time = ''.join(np.fromfile(fid, dtype='S1', count=12))
+        date = session_date.split('/')
+        if date[2].startswith('9'):
+            date[2] = '19' + date[2]
+        elif len(date[2]) == 2:
+            date[2] = '20' + date[2]
```

Comment:
looks like there should be a cleaner way to do this. Maybe https://docs.python.org/2/library/time.html#time.strftime ?